src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* BUGFIX: Fix Error BA2006: '...' was compiled with one or more modules which were not built using minimum required tool versions [545](https://github.com/microsoft/binskim/pull/545)
```

Comment:
>Fix Error BA2006: '...' was compiled with one or more modules which were not built using minimum required tool versions [](http://example.com/codeflow?start=10&length=119)

We are fixing the incorrect analysis for compilers that aren't Microsoft.

Maybe something like: "Fix incorrect analysis for non-Microsoft compiler on BA2006.RuleName."

Reply:
agree, this is both short and clear.